actions/setup-tfenv-terraform/action.yml

```diff
@@ -12,7 +12,9 @@ runs:
         if tfenv --version; then
           echo "tfenv already installed, skipping"
         else
-          git clone --depth=1 https://github.com/tfutils/tfenv.git ~/.tfenv
+          git clone --depth=1 https://github.com/tfutils/tfenv.git temp
+          mv temp ~/.tfenv
+          rm -rf temp
```

Comment:
Hmm now that I look at this (and the answer at https://stackoverflow.com/a/13852329/589391), I can think of no need for the `rm -rf temp` line since the `mv` will remove that directory.

Reply:
And, ugh, sorry to add more to this, but to make sure we avoid any conflicts in the directory where this is running, we should really do this:

```
TMPDIR=$(mktemp -d)
git clone --depth=1 https://github.com/tfutils/tfenv.git $TMPDIR
mv $TMPDIR ~/.tfenv
```